tests/stub/retry/scripts/clustering/router_retry_once.script

```diff
@@ -0,0 +1,13 @@
+!: BOLT #VERSION#
+!: AUTO RESET
+!: AUTO GOODBYE
+!: ALLOW RESTART
+
+C: HELLO {"scheme": "basic", "credentials": "c", "principal": "p", "user_agent": "007", "routing": #HELLO_ROUTINGCTX# #EXTRA_HELLO_PROPS#}
+S: SUCCESS {"server": "Neo4j/4.0.0", "connection_id": "bolt-123456789"}
+{*
+    C: ROUTE #ROUTINGCTX# [] null
+    # TODO: double-check if that routing table works with other drivers (may need to
+    # add extra roles)
+    S: SUCCESS { "rt": { "ttl": 1000, "servers": [{"addresses": ["#HOST#:9003"], "role":"WRITE"}]}}
```

Comment:
```suggestion
    C: ROUTE #ROUTINGCTX# [] null
    S: SUCCESS { "rt": { "ttl": 1000, "servers": [{"addresses": ["#HOST#:9001"], "role":"ROUTER"}, {"addresses": [], "role":"READER"}, {"addresses": ["#HOST#:9003"], "role":"WRITE"}]}}
```
Is a more realistic RT you'd seem from servers. Even though a server would never be shown as writer but not as reader if I'm not mistaken.

Reply:
That's not what happens with SSR though, so if that's not an issue with other drivers, I'd prefer to keep the RT with a single entry